src/PresentationalComponents/RulesTable/RulesTable.spec.ct.js

```diff
@@ -0,0 +1,642 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { mount } from '@cypress/react';
+import RulesTable from './RulesTable';
+import { Provider } from 'react-redux';
+import { IntlProvider } from '@redhat-cloud-services/frontend-components-translations/';
+import { getStore } from '../../Store';
+import fixtures from '../../../cypress/fixtures/rulesfixtures';
+import _ from 'lodash';
+
+// eslint-disable-next-line rulesdir/disallow-fec-relative-imports
+import {
+  checkTableHeaders,
+  checkPaginationTotal,
+  checkPaginationValues,
+  PAGINATION_VALUES,
+  TOOLBAR,
+  tableIsSortedBy,
+  hasChip,
+  CHIP_GROUP,
+  changePagination,
+  checkRowCounts,
+  itemsPerPage,
+  PAGINATION,
+  PAGINATION_MENU,
+  DROPDOWN_TOGGLE,
+  SORTING_ORDERS,
+  checkSorting,
+  removeAllChips,
+  applyFilters,
+  filter,
+  checkNoMatchingRecs,
+  TOOLBAR_FILTER,
+  TITLE,
+  TABLE,
+  checkFiltering,
+} from '@redhat-cloud-services/frontend-components-utilities';
+
+//I'm looking at the https://docs.cypress.io/guides/component-testing/custom-mount-react#React-Router
+
+import { createIntl, createIntlCache } from 'react-intl';
+import { cellWidth, sortable, fitContent } from '@patternfly/react-table';
+import messages from '../../Messages';
+//declaring intl to provie it to the columns
+const cache = createIntlCache();
+const intl = createIntl(
```

Comment:
perhaps we could use this https://github.com/RedHatInsights/insights-advisor-frontend/blob/master/src/AppConstants.js#L11?

Reply:
Don't really understand what you meant by that.

Reply:
@Fewwy , I mean this looks like a duplicate code and, in the file I sent, there is already an `intl` instantiated. We could just make an export of it and then import in your tests